Dockerfile

```diff
@@ -1,12 +1,16 @@
 FROM node:8
 
+# specify "dev" or "prod"
+ARG target=prod
+
 WORKDIR /app
 
 COPY . ./
 
 RUN yarn
 
-RUN yarn build
+RUN echo "Target is set to: $target"
```

Comment:
Each invocation of `RUN` creates a separate layer in the Docker image.

The `target` environment is already printed to the stdout when building the image:
```
Building orca-ui
Step 1/9 : FROM node:8
 ---> 8eeadf3757f4
Step 2/9 : ARG target=prod
```
I think the `echo` step can be removed to reduce the layer overhead (minor but needed).

Reply:
I removed this step.

I think that in future, we should add a proper, production Dockerfile utilizing multi-stage building (I will create an issue for this) and separate development Dockerfile.